recipes/expat/all/conanfile.py

```diff
@@ -106,8 +90,11 @@ def package_info(self):
         self.cpp_info.set_property("cmake_target_name", "expat::expat")
         self.cpp_info.set_property("pkg_config_name", "expat")
 
+        self.cpp_info.filenames["cmake_find_package"] = "EXPAT"
+        self.cpp_info.filenames["cmake_find_package_multi"] = "expat"
         self.cpp_info.names["cmake_find_package"] = "EXPAT"
         self.cpp_info.names["cmake_find_package_multi"] = "expat"
+        self.cpp_info.names["pkg_config_name"] = "expat"
```

Comment:
there is no `pkg_config_name` in legacy `cpp_info.names`, it's `pkg_config`, but it's useless anyway since legacy `pkg_config` inherits from new `pkg_config_name` property.